api/rest/v1/bulk_find_entities.md

```diff
@@ -0,0 +1,157 @@
+</div>
+
+<div markdown="span" class="alert alert-info" role="alert">
+   <span class="material-icons md-16">info </span><b>Note:</b><br />
+   Currently, this endpoint only supports [place](/glossary.html#place) entities.
+</div>
+
+## Request
+
+POST Request
+{: .api-header}
+
+<div class="api-signature">
+URL:
+https://api.datacommons.org/v1/bulk/find/entities
+
+Header:
+X-API-Key: {your_api_key}
+
+JSON Data:
+{
+  "entities": [
+    {
+        "description": "{entity_name_1}",
+        "type": "{entity_type_1}"
+    },
+    {
+        "description": "{entity_name_2}",
+        "type": "{entity_type_3}"
+    },
+    ...
+  ]
+}
+</div>
+<script src="/assets/js/syntax_highlighting.js"></script>
+
+### Path Parameters
+
+There are no path parameters for this end point.
+
+### Query Parameters
+
+| Name                                                     | Type   | Description                                                                                                                                                     |
+| -------------------------------------------------------- | ------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| key <br /> <required-tag>Required</required-tag>         | string | Your API key. See the [page on authentication](/api/rest/v1/getting_started#authentication) for a demo key, as well as instructions on how to get your own key. |
+| description <br /> <required-tag>Required</required-tag> | string | The name of the entity.                                                                                                                                         |
+| type <br /> <optional-tag>Optional</optional-tag>        | string | The DCID of the entity's class. Common values are "Country", "State", "County", "City".                                                                         |
+{: .doc-table }
+
+## Response
+
+The response looks like:
+
+```json
+{
+  "entities": [
+    {
+      "description":"Description provided",
+      "type":"Type provided",
+      "dcids":["DCID"]
+    }
+  ]
+}
+```
+{: .response-signature .scroll}
+
+### Response fields
+
+| Name        | Type   | Description                                                                                            |
+| ----------- | ------ | ------------------------------------------------------------------------------------------------------ |
+| description | string | The description you provided.                                                                          |
+| type        | string | The DCID of the entity's class, if provided.                                                           |
```

Comment:
This is "type", not the "DCID"?

Reply:
Changed the wording to clarify, but wanted to make sure users know to provide the DCID of the class. For example, `type: "country"`(lowercase c) does not return matches while `type: "Country"` (uppercase C) does.